README.md

```diff
@@ -50,18 +49,17 @@ for more information. See also the [`AUTHORS`](AUTHORS) and
 - `third_party/`: third party open source packages; see below
 - `utils/`: utility scripts for Shaderc
 
-Shaderc depends on `glslang`, the Khronos reference compiler for GLSL.
+Shaderc depends on glslang, the Khronos reference compiler for GLSL.
 Sometimes a change updates both Shaderc and glslang.  In that case the
-glslang change will appear in [google/glslang](https://github.com/google/glslang)
-before it appears upstream in
-[KhronosGroup/glslang](https://github.com/KhronosGroup/glslang).
+glslang change will appear in [google/glslang][google-glslang]
+before it appears upstream in [KhronosGroup/glslang][khr-glslang]
 We intend to upstream all changes to glslang. We maintain the separate
 copy only to stage those changes for review, and to provide something for
 Shaderc to build against in the meantime.  Please see
 [DEVELOPMENT.howto.md](DEVELOPMENT.howto.md) for more details.
 
-Shaderc depends on [SPIRV-Tools](https://github.com/KhronosGroup/SPIRV-Tools)
-for assembling and disassembling SPIR-V binaries.
+Shaderc depends on [SPIRV-Tools][spirv-tools] for assembling and disassembling
+SPIR-V binaries.
```

Comment:
"assembling, disassembling, and transforming SPIR-V binaries."

(we use it to strip debug info, for example).